src/components/ClaimReview/UserAutocomplete.tsx

```diff
@@ -1,28 +1,39 @@
-import React from 'react'
+import React from "react";
 import api from "../../api/user";
-import { useTranslation } from 'next-i18next'
-import SelectUser from './form/SelectUser'
+import { useTranslation } from "next-i18next";
+import SelectUser from "./form/SelectUser";
 
-const UserAutocomplete = ({ placeholder, onChange, dataCy }) => {
+const UserAutocomplete = ({ placeholder, onChange, dataCy, fieldName }) => {
     const { t } = useTranslation();
 
     const fetchUserList = async (name) => {
```

Comment:
Renaming the component to Autocomplete means that we should pass the `fetchUserList` or the `fetchTopicList` as "data loaders", adding a new prop to the component `dataLoader`